one_liners.js

```diff
@@ -0,0 +1,172 @@
+  'Four of your five wits went halting off, and now is %u% governed with one.', // Beatrice, Much Ado About Nothing
+  '%u%, freaking idiot.', // Napolean, Napolean Dynomite
+  'There are some people you like immediately, some whom you think you might learn to like in the fullness of time, and some that you simply want to push away from you with a sharp stick, %u%.', // Douglas Adams, The Long Dark Tea Time of the Soul
+  '%u%, Copernicus just called: You’re not the center of the universe.', // Anonymous
+  'The only decent impression %u% can do is of the man with no talent.', // Blackadder, Blackadder
+  'Hab SoSlI’ Quch, %u%!', // Anonymous, Klingon
+  'You are a driver, %u% — and I use the word in the loosest possible sense, i.e., meaning merely somebody who occupies the driving seat of what I will for the moment call — but I use the term strictly without prejudice — a car while it is proceeding along the road — of stupendous, I would even say verging on the super-human, lack of skill.', // Dirk Gently, The Long Dark Tea Time of the Soul
+  'Never hung poison on a fouler toad. Out of my sight! Thou dost infect mine eyes, %u%.', // Anne, Richard III
+  'You clinking, clanking, clattering collection of caliginous junk, %u%!', // The Wizard, The Wizard of Oz
+  '%u% is about as effective as a cat-flap in an elephant house.', // Blackadder, Blackadder
+  'You don’t have the brains to understand, %u%. All you have is printed circuits.', // Captain Kirk, Star Trek
+  'If brains were photons you would be perfect for the double slit experiment, %u%.', // Anonymous
+  'Look, %u%, we all have something to bring to this discussion. But I think from now on the thing you should bring is silence.', // Arnold Rimmer, Red Dwarf
+
+  // some nice Unicode art
+  '凸(-_-)凸',
+  '╭∩╮(-_-)╭∩╮',
+  '(╯°□°）╯︵ ┻━┻',
+];
+
+var stickers = [
+  'BQADBAADIAADyIsGAAGeqFpovvSWiwI', // Julius Caesar 👎
+  'BQADBAADPwADyIsGAAFyYVwK5nqWFQI', // Elvis Presley 😂
+  'BQADBAADLQADyIsGAAE_-arlvGeRjgI', // Jay Hawkins 😲
+  'BQADBAADMAADyIsGAAHU8vIAAev_v-UC', // Dante Alighieri 😖
+  'BQADBAADFQADyIsGAAEO_vKI0MR5bAI', // Nikola Tesla 😔
+  'BQADAgAD5gAD9HsZAAE3KspwHpaxlwI', // 🐨
+  'BQADAgAD6gAD9HsZAAFDRbdAUmM_jQI', // 🐻 Facepalm
+  'BQADAgAD-AAD9HsZAAGMoglB6izVgwI', // 🐱 OMG
+  'BQADAgADQwEAAvR7GQABzeqCC-X7ZYQC', // 👀 Seal
+  'BQADAgADYgEAAvR7GQABNgdi7-hwi4gC', // ✋ Hedgehog Middlefinger
+  'BQADBAADOQADmu78Ap0s66P75zWsAg', // Toni Brunner 😮
+  'BQADBAADRwADmu78AklGZ2FcPSkKAg', // Ueli Maurer 😀
+  'BQADBAADrQADEhKdAAG75Oi8qTO6fwI', // 👐 Chabis
+  'BQADBAADwQADEhKdAAFk3LSH1pscvgI', // 😈 Verzells am Chääs
+];
+
+/**
+ * Returns a random insult
+ *
+ * @param {string} userName - The user's name who should be insulted
+ * @returns {string} A random insult
+ */
+exports.getRandomInsult = function(userName) {
+  var insult = insults[Math.floor(Math.random() * insults.length)];
+  return insult.replace('%u%', userName);
```

Comment:
Bug: `'%u%'` has to be `/%u%/g` to replace all concurrencies of %u% in the string.